stdlib/test/collections/test_list.mojo

```diff
@@ -571,6 +571,19 @@ def test_constructor_from_other_list_through_pointer():
     assert_equal(some_list.capacity, capacity)
 
 
+def test_converting_list_to_string():
```

Comment:
**Suggestion** Perhaps add a test case for nested lists?  E.g. `List[List[Int]]`.

Reply:
I can't do this, because `List[Int]` is not Stringable, even after this PR :(